test/integration_test_helper.rb

```diff
@@ -28,8 +29,19 @@
   Capybara::Poltergeist::Driver.new(app, opts)
 end
 
-Capybara.default_max_wait_time = 30
-Capybara.javascript_driver = :poltergeist
+Selenium::WebDriver::Chrome.driver_path = File.join(Rails.root, 'node_modules', '.bin', 'chromedriver')
+Capybara.register_driver :selenium_chrome do |app|
+  capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(
+    chromeOptions: { args: %w(headless disable-gpu) }
+  )
+
+  Capybara::Selenium::Driver.new app,
+    browser: :chrome,
+    desired_capabilities: capabilities
+end
+
+Capybara.default_max_wait_time = 5
```

Comment:
won't this change the current default for phantomjs too? I would suggest to keep phantomjs on our jenkins slaves, but enable developers to use the faster chrome driver.

Reply:
This is intentional. The lower wait time is to try and see if we can have this lowered.

Locally I had cases where tests seemed to simply waiting that time, idle, taking about 30 seconds and then pass. The lower wait time made those faster and still pass.